spec/unit/lib/temporal/grpc_spec.rb

```diff
@@ -39,6 +39,7 @@ class TestDeserializer
           run_timeout: 0,
           task_timeout: 0,
           memo: {},
+          search_attributes: {},
```

Comment:
It would be great to have at least one test example here that actually checks that search attributes are getting serialized properly